integration/shared/isolated/disable_service_access_command_test.go

```diff
@@ -156,12 +156,12 @@ var _ = Describe("disable service access command", func() {
 						session = helpers.CF("service-access", "-e", service)
 						Eventually(session).Should(Exit(0))
 						Eventually(session).Should(Say("broker:\\s+%s", broker.Name))
-						Eventually(session).Should(Say("%s\\s+%s\\s+none",
+						Eventually(session).Should(Say("%s\\s+.*\\s+all",
```

Comment:
Two spaces will match the regex `\\s+.*\\s+`. It seems a little too loose. We're wondering why the expectation needed to be loosened in the first place. If it does need to be loosened, we should probably change the `.*` to a `.+`.